x-pack/plugins/reporting/common/constants.ts

```diff
@@ -5,7 +5,6 @@
  * 2.0.
  */
 
-// Export Type Sets
 import {
```

Comment:
🤞🏻

Reply:
The issue was that moving the job types to @kbn/reporting-common caused issues because that package supplies information to the png/pdf/csv-common packages. I think that having the constants in the plugin is ok since it's the end of the funnel for all the packages anyways, wdyt?